### PR TITLE
Implement Signup Page

### PR DESCRIPTION
**Pull request by Sanjay-B**
This feature implements the signup page. All inputs work but, are not hooked into any backend technologies.
 
http://localhost:3000/signup

### CODE REVIEW

File: src/pages/signup.js [hunk 1]
```diff
@@ -10,34 +10,39 @@ export default function Signup() {
...
-                <Grid container spacing={2}>
-                    <Grid item xs={12}>
-                        <TextField required fullWidth id="full_name" label="Full Name" name="full_name" autoComplete="full name" />
+        <Container component='main' maxWidth='xs'>
+            <Box sx={{ marginTop: 8, display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+                <Typography variant='h5'>Sign up</Typography>
+                <Box component='form' noValidate onSubmit={handleSubmit} sx={{ mt: 3 }}>
+                    <Grid container spacing={2}>
+                        <Grid item xs={12}>
+                            <TextField required fullWidth id="full_name" label="Full Name" name="full_name" autoComplete="full name" />
```
Comment (vhorvath2010) [inline src/pages/signup.js:19]: Please break this down into first + last name so we can easily address attendees as such if we choose to send email communications

**Review by vhorvath2010 — CHANGES_REQUESTED** [review]
Please break down the full-name field into first name and last name so we can address attendees with one or the other if we choose to automatically email them.

**Comment by vhorvath2010** [comment]
Also, please make it so only PDFs can be submitted as resumes (adding this field to the input tag)
`accept=".pdf"`

**Comment by Sanjay-B** [comment]
> Please break down the full-name field into first name and last name so we can address attendees with one or the other if we choose to automatically email them.

> Also, please make it so only PDFs can be submitted as resumes (adding this field to the input tag) `accept=".pdf"`

Fixed as of https://github.com/KSU-Google-DSC/Google-Summit-Solution/pull/25/commits/c21c6f5e282c8fe63cb259cd87c4d5d5e3885144. @vhorvath2010

**Pull request merged**